tests/conftest.py

```
@@ -0,0 +1,28 @@
+import os
+import pytest
+from sqlalchemy import create_engine
+from sqlalchemy.orm import scoped_session, sessionmaker
```

Ruff (0.11.9)

`sqlalchemy.orm.scoped_session` imported but unused

Remove unused import: `sqlalchemy.orm.scoped_session`

(F401)